### DISCUSSION TITLE
Suggestion: Sprite Layer Manager Tab

### DISCUSSION THREAD

**Discussion by tokosaucek** [ScratchAddons/ScratchAddons#4745] [Ideas]
Suggesting a tab for managing sprite layers similar to the variable manager tab addon where you can sort sprites' layers on the stage (potentially by clicking and dragging them in order like costumes or sounds). This would make it easier to determine where sprites are on the stage without having to use the `move up/down ( ) layers` or `move to top/bottom layer` block. Thank you. 

> <img width="237" alt="screenshot of the code, costumes, sounds, sounds, and variables tabs" src="https://user-images.githubusercontent.com/62413702/176995021-55ba1106-2415-4512-b33a-e9e15d188a20.png">
> 
> ^ Something like one of these

**Comment by Norbiros** [comment]
#4722
